src/coreclr/src/jit/patchpoint.cpp

```diff
@@ -185,6 +207,41 @@ class PatchpointTransformer
 
         compiler->fgNewStmtNearEnd(block, ppCounterAsg);
     }
+
+    //------------------------------------------------------------------------
+    // TransformUncommon: expand current block to include uncommon patchpoint logic.
+    //
+    //  S;
+    //
+    //  ==>
+    //
+    //  call JIT_UNCOMMON_PATHCPOINT (ilOffset)
```

Comment:
I don't understand this method header comment.

Reply:
It's trying to say that we delete all the statements in the block and add a call; I'll clarify.